src/Heading.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const Heading = () => (
+  <header className="App-header">
```

Comment:
Good job extracting the heading!

One small suggestion; the most common convention for naming css classes is to use all lower case and have `-` between words. Most of your classes in `App.css` follow this convention except for the two classes used here. I would change them to `app-header` and `app-logo` so your code is consistent.